example/e2e/detoxTest.spec.js

```diff
@@ -154,7 +175,7 @@ describe('Example', () => {
       await userChangesMinuteValue();
       await userTapsOkButtonAndroid();
 
-      await expect(timeText).toHaveText('22:30');
+      await expect(timeText).toHaveText('23:30');
```

Comment:
what this PR does is that it adds a new feature, and yet it changes old test behavior 🤔  - either there was a bug before and the test was wrong, or this PR introduced a bug. I tend to believe the second is true

compare `setTz should change time text when time changes 60 minutes` with `should change time text when time changes`

Reply:
It is becuase I changed order of the app settings.